frontend/src/views/pages/MainPage.tsx

```diff
@@ -1,49 +1,87 @@
-// src/App.tsx
-
 import React, { useEffect, useState } from "react";
 import "./MainPage.css";
 import Footer from "../components/Footer/Footer";
 import HeroSection from "../components/HeroSection/HeroSection";
 import ProcessStep from "../components/ProcessStep/ProcessStep";
 import Table from "../components/Table/Table"
-import logo from '@/assets/Square_Cash_app_logo.png'; 
-const apiUrl = import.meta.env.VITE_API_URL;
+import Title from "../components/Title/Title";
+import GraphImage from "../assets/GraphImage.png"
+import DownArrow from "../assets/DownArrow.png"
+import Header from "../components/Header/Header";
 
 const MainPage: React.FC = () => {
 
   return (
-    <div className="MainPage">
-      <div className="App">
-      <div className="sticky-logo">
-        <img src={logo} alt="Logo" />
-      </div>
-      {/* Hero Section */}
-      <HeroSection
-        heading="Target fraud detection bias at the source."
-        body="Some sort of subtitle text here. Check out our handy guide ↓ if you need any help!"
-      />
 
-      {/* Process Step / Instruction Section */}
-      <ProcessStep
-        title="Upload your dataset."
-        stepnum={1}
-        body="The format of the dataset must be a parquet file with the following mandatory columns highlighted in the table below."
-      />
-      <Table />
-      <ProcessStep
-        title="View our feedback."
-        stepnum={2}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
-      <ProcessStep
-        title="Adjust and try again!"
-        stepnum={3}
-        body="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean molestie mi id sapien posuer."
-      />
+    <div className="main-page">
+      <Header />
+
+      <Title label="BYE BYE BIAS"/>
 
+      <div className="arrow-container">
```

Comment:
I think this needs to be a button for accessibility, also if its a button, maybe remove the alt=Down Arrow since that's a decorative img (no alt text)

Reply:
makes sense! - this will be addressed in Chloe's pr